user_groups/user_groups.md

```diff
@@ -0,0 +1,294 @@
+    "name": "Project Team"
+  }
+]
+```
+---
+
+## Get a user group
+
+Returns a user group at a given id.
+
+```apacheconfig
+GET /v1/userGroups/{userGroupId}
+```
+
+#### Path parameters
+
+| Name | Type | Description |
+| ------------- |------------- | -------------|
+| userGroupId | `string` | The id of a user group to request. |
+
+
+#### Query parameters
+The query parameters are not expected.
+
+#### Request body
+The request body must be empty.
+
+#### Response body
+The successful response contains an instance of a [UserGroup](./user_groups_schemas.md/#user-group).
+
+The error response contains an [Error](../response_codes.md).
+
+### Example
+
+Request
+
+```shell
+curl https://app.solvexia.com/api/v1/userGroups/ug-114273 -X GET -H "Authorization: Bearer syPHeMY5H--kdRtfpoXTgYFF7LHgVOhIjOQ5QkIvSD68VZvc2_uAew.P07tEVThD5SqNCV_tFwbAg"
+```
+
+Response
+
+```json
+{
+    "id": "ug-114273",
+    "name": "Sales Team",
+    "description": "Description for a user group"
+}
+```
+---
+
+## Get members for a user group
+
+Get a list of members for a user group.
+
+```apacheconfig
+GET /v1/userGroups/{userGroupId}/members
+```
+
+#### Path parameters
+
+| Name | Type | Description |
+| ------------- |------------- | -------------|
+| userGroupId | `string` | The user group id to request. |
+
+#### Query parameters
+The query parameters are not expected.
+
+#### Request body
+The request body must be empty.
+
+#### Response body
+The successful response contains an array of instances of a [User](../users/users_schemas.md/#user).
+
+The error response contains an [Error](../response_codes.md).
+
+### Example
+
+Request
+
+```shell
+curl "https://app.solvexia.com/api/v1/userGroups/ug-114273/members" -X GET -H "Authorization: Bearer syPHeMY5H--kdRtfpoXTgYFF7LHgVOhIjOQ5QkIvSD68VZvc2_uAew.P07tEVThD5SqNCV_tFwbAg"
+```
+
+Response
+
+```json
+[
+  {
+    "id": "u-1233",
+    "loginName": "template.tom.jordan"
+  },
+  {
+    "id": "u-1463",
+    "loginName": "template.jack.mayer"
+  }
+]
+```
+---
+
+## Get user group permission
+
+Get a list of user group permissions.
+
+```apacheconfig
+GET /v1/userGroups/{userGroupId}/permissions
+```
+
+#### Path parameters
+
+| Name | Type | Description |
+| ------------- |------------- | -------------|
+| userGroupId | `string` | The user group id to request. |
+
+#### Query parameters
+The query parameters are not expected.
+
+#### Request body
+The request body must be empty.
+
+#### Response body
+The successful response contains a list of [Permission](./user_groups_schemas.md/#permission).
+
+The error response contains an [Error](../response_codes.md).
+
+### Example
+
+Request
+
+```shell
+curl "https://app.solvexia.com/api/v1/userGroups/ug-114273/permisions" -X GET -H "Authorization: Bearer syPHeMY5H--kdRtfpoXTgYFF7LHgVOhIjOQ5QkIvSD68VZvc2_uAew.P07tEVThD5SqNCV_tFwbAg"
+```
+
+Response
+
+```json
+[
+  {
+    "resourceId": "p-2343",
+    "resourceName": "Sales reconciliation",
+    "role": "executor"
```

Comment:
Here the executor starts with a small "e" where is in the Permissions enum it's the capital "E". We need to make it consistent.

Reply:
using small letter everywhere for this enum now